addons/van_dml/example/van_dml.js

```diff
@@ -0,0 +1,67 @@
+/* van_dml: add auto-append to the van.tags
+    elements are added automatically to the selected object between begin and end
+    begin(document.body)
+        h1("test it")
+    end
+    SP is the stackPointer, should be 0 at end of JS
+    base is the current base object for append, null if none
+*/
+(() => {
+    let _base = null;                // current inser point
+    const _baseStack = [];    // Stack for storing _base positions
+    const orgtags = van.tags
+    // select new base for append, previous will be saved
+    function begin(ID) {
+        _baseStack.push(_base);// Save old base
+        if (_baseStack.length > 100) throw new Error("_baseStackOverflow in begin()");
+        _base = (typeof (ID) === 'string') ? document.getElementById(ID) : ID
+        return _base
+    }
+    // restore last base, cnt: call multiple times
+    function end(cnt = 1) {
+        if (cnt > 1) end(cnt - 1)
+        if (_baseStack.length <= 0) throw new Error("_baseStack empty in end()")
+        return (_base = _baseStack.pop())    // restore old stack
+    }
+
+    // Proxy the proxy
+    const tags = new Proxy((f, ...args) => {
+        let r = f(...args)
+        if (_base) van.add(_base, r)
+        return r
+    }, {
+        get: (target, name) => {
+            let o = {}; o[name] = 0; o = orgtags
+            return target.bind(null, o[name])// o[name]
+        }
+    });
+
+    /* ----------------------------------------------------------------------------
+    CSS: create CSS-Defintions dynamically
+    s can be 
```

Comment:
s can be wrapped by ``

Reply:
I changed the text to make it more clear.

Reply:
I mean, s can be wrapped by "``".

Reply:
Ok, did it